securityscanutils/README.md

```diff
@@ -52,6 +52,8 @@ func main() {
                     // endpoint, e.g. https://github.com/solo-io/gloo/security/code-scanning
                     // read more here: https://docs.github.com/en/rest/reference/code-scanning
 					UploadCodeScanToGithub: true,
+					// Opens/Updates Github Issue for each image a vulnerability is found in
```

Comment:
Formatting looks off here?

Reply:
This is how the fomratting looks in my IDE:
![image](https://user-images.githubusercontent.com/1731122/126835650-cad34fdf-ba1a-4063-b8dd-21adb3069184.png) even after pulling from this branch. Might be a GH rendering issue?